dlc-oracle/src/main/scala/org/bitcoins/dlc/oracle/storage/SeedStorage.scala

```diff
@@ -0,0 +1,243 @@
+package org.bitcoins.dlc.oracle.storage
+
+import java.nio.file.{Files, Path}
+import java.time.Instant
+import java.util.NoSuchElementException
+
+import org.bitcoins.core.crypto.ExtKeyVersion.SegWitMainNetPriv
+import org.bitcoins.core.crypto.{BIP39Seed, ExtPrivateKey}
+import org.bitcoins.crypto._
+import org.bitcoins.keymanager.{DecryptedMnemonic, EncryptedMnemonic}
+import org.slf4j.LoggerFactory
+import scodec.bits.ByteVector
+
+import scala.util.{Failure, Success, Try}
+
+object SeedStorage {
```

Comment:
Is there a reason we don't just depend on the key manager project and use that code? I think key manager is zero dep

Reply:
The reason I did this is so I can get direct access to the `ExtPrivateKey`. Using whats in the km project, we don't have direct access to get `ExtPrivateKey` and thus can't get the keys on the exact paths I want.

Reply:
Hmm... ok this is a bit worrying for reasons discussed this morning (https://github.com/discreetlogcontracts/dlcspecs/issues/90). I think one thing that would make me feel better about people not foot gunning this and just using `ExtPrivateKey` directly and deriving a bunch of non hardened keys would be creating a new type -- let's say `ExtKeyHardened` that allows ONLY hardened derivation. 

...also is it necessarily to copy paste everything over? We should probably at least attempt to re-use some code so we don't have to maintain two different things that do more or less the exact same thing. Also it's very sensitive code, so the less copy-paste the better I would feel about this. Then when we make chagnes to the key manager project this release we won't forget about this project.

Reply:
I like this idea of a `ExtKeyHardened`, however, I think I will do that in a separate PR onto `master`